### PR TITLE
Feat/kill uib accordion

### PR DESCRIPTION
**Pull request by radireddy**
Replace uib-accordion with oui-collapsible

#MANAGER-4049

### CODE REVIEW

File: packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/queues/ovhPabx/telecom-telephony-alias-configuration-queues-ovhPabx.html [hunk 1]
```diff
@@ -35,239 +35,229 @@ <h1 data-translate="telephony_alias_configuration_queues_title"></h1>
...
-                               data-ng-class="{'glyphicon-chevron-up': queue.isOpen, 'glyphicon-chevron-down': !queue.isOpen}"></i>
-                        </div>
-                    </uib-accordion-heading>
+            <oui-collapsible
+                data-ng-repeat="queue in QueuesOvhPabxCtrl.queues track by queue.queueId"
+                data-heading="{{ ::queue.description ?
+                    ('telephony_alias_configuration_queues_queue' | translate) + queue.description :
+                    ('telephony_alias_configuration_queues_queue' | translate) + '#' + queue.queueId
+                    }}"
+                data-expanded="queue.isOpen ? true : false"
```
Comment (JeremyDec) [inline packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/queues/ovhPabx/telecom-telephony-alias-configuration-queues-ovhPabx.html]: Why not directly ?
```suggestion
                data-expanded="queue.isOpen"
```

File: packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/queues/ovhPabx/telecom-telephony-alias-configuration-queues-ovhPabx.html [hunk 2]
```diff
@@ -35,239 +35,229 @@ <h1 data-translate="telephony_alias_configuration_queues_title"></h1>
...
+                        class="btn btn-danger float-right clearfix"
+                        data-ng-click="QueuesOvhPabxCtrl.deleteQueue(queue)"
+                        data-ng-disabled="QueuesOvhPabxCtrl.isDeleting || QueuesOvhPabxCtrl.queues.length <= 1">
+                    <oui-spinner class="mr-2"
+                                data-ng-if="QueuesOvhPabxCtrl.isDeleting"
+                                data-size="s">
+                    </oui-spinner>
+                    <span data-translate="telephony_alias_configuration_queues_queue_delete"></span>
+                </button>
+                <div class="clearfix"></div>
```
Comment (FredericEspiau) [inline packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/queues/ovhPabx/telecom-telephony-alias-configuration-queues-ovhPabx.html]: Are both `clearfix` really needed ? Can't you replace with `flex` for example ?

**Comment by antleblanc** [comment]
Hi @radireddy!

We recently merged #1336 which apply Prettier rules so could you please rebase
your branch on `develop`?

To easily resolve conflicts you can **ignore** `HEAD` changes and only keep yours.

After that you can run:

```sh
$ yarn run lint
```

This command will lint the entire codebase.

or:

Amend your commit to trigger the `pre-commit` hook (prettier will be launched
under the hood).

Thanks!

**Comment by antleblanc** [comment]
ping @radireddy!

Could you please confirm if it's still bocked?
And then resolve existing conflicts.

Thanks!

**Comment by antleblanc** [comment]
@radireddy Pull Request has been inactive for a long period so I will close it.
Feel free to reopen it if needed.

Thank you!

**Pull request closed without merging**